playbooks/roles/figures/tasks/main.yml

```diff
@@ -56,15 +56,25 @@
     - figures
     - deploy:figures
 
-- name: install figures from {{ figures_code_dir }}
+- name: install figures {{ figures_version }} from {{ figures_code_dir }}
   args:
     chdir: "{{ figures_code_dir }}"
   command: "{{ edxapp_venv_dir }}/bin/pip install -e ."
   become_user: "{{ edxapp_user }}"
   environment: "{{ edxapp_environment }}"
   tags:
     - figures
-    - install:figures
+    - figures:install
+
+- name: migrate figures
```

Comment:
Isn't figures supposed to just migrate alongside with the LMS? Why do we need a separate step?

Reply:
@OmarIthawi updated the comments for the migrate step. Hope this clears up. 
cc @thraxil